axelor-stock/src/main/resources/i18n/messages_de.csv

```diff
@@ -401,9 +400,11 @@
 "Partner product quality ratings","Bewertungen der Produktqualität durch Partner",,
 "Partner stock settings","Partnerbestandseinstellungen",,
 "Perishable expiration date","Verfallsdatum der verderblichen Ware",,
+"Picking printed","Bon de prépa édité",,
```

Comment:
Why french translation appears here ?

Reply:
It's an ADK issue.
It was automatically generated on I18n build.